test/spec/MfaVerify_spec.js

```diff
@@ -241,21 +244,32 @@ function (Q, _, $, Duo, OktaAuth, LoginUtil, CryptoUtil, Util, MfaVerifyForm, Be
             Expect.isVisible(test.form.rememberDeviceCheckbox());
           });
         });
-        itp('has the right text', function () {
-          return setup(resAllFactors).then(function (test) {
-            expect(test.form.rememberDeviceLabelText()).toEqual('Trust this device');
+        itp('has the right text for device based policy', function () {
+          return setup(resMfaDevicePolicy).then(function (test) {
+            expect(test.form.rememberDeviceLabelText()).toEqual('Do not challenge me on this device again');
+          });
+        });
+        itp('has the right text for time based policy', function () {
+          return setup(resMfaTimePolicy).then(function (test) {
+            expect(test.form.rememberDeviceLabelText()).toEqual(
+              'Do not challenge me on this device for the next 15 minutes');
           });
         });
```

Comment:
Can we also add two tests - one for 1 min and the other for hours/days to test getMinutesString() function?